system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -1935,6 +1937,10 @@ class CLdapClient : implements ILdapClient, public CInterface
         if(infotype && stricmp(infotype, "sudoers") == 0)
         {
             CLdapSecUser* ldapuser = dynamic_cast<CLdapSecUser*>(&user);
+            if (ldapuser == nullptr)
+            {
+                throw MakeStringException(-1, "Unable to cast input user variable");
```

Comment:
I can't provide the user because it's just an ISecUser at this point, and I am unable to cast it to a class where I can extract values.